eth/tracers/native/call.go

```diff
@@ -166,5 +166,6 @@ func uintToHex(n uint64) string {
 }
 
 func addrToHex(a common.Address) string {
-	return strings.ToLower(a.Hex())
+	s, _ := a.MarshalText()
+	return strings.ToLower(string(s))
```

Comment:
Why call `strings.ToLower` ? I think return `string(s)` is enough:

- We should keep the address is checksumd
- reduce CPU cost and network response time

Reply:
Ok. Ethereum uses lower case. If we don't need lower case we can just return the string(s)